c2corg_ui/templates/waypoint/view.html

```diff
@@ -154,16 +154,18 @@ <h3 class="heading informations" ng-click="mainCtrl.animateHeaderIcon($event)" d
         </section>
       </span>
     </div>
-    <div class="view-details-associations col-xs-12  associations" ng-if="detailsCtrl.documentService.document.associations.all_routes.total > 0">
+    <div class="view-details-associations col-xs-12 associations">
       <span class="lead">
         <section>
           ${show_associated_routes(waypoint, 'all_routes')}
         </section>
       </span>
     </div>
 
-    % if 'recent_outings' in waypoint['associations'] and waypoint['associations']['recent_outings']['total'] > 0:
-      <div class="view-details-associations col-xs-12  associations">
+    % if waypoint['associations']['recent_outings']['total'] or \
```

Comment:
You're definitely right! I have replaced this change by an ``ng-show`` condition as you suggest it. See a4244bc